client/my-sites/site-settings/form-general.jsx

```diff
@@ -330,6 +330,47 @@ class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	netNeutralityOption() {
+		const { fields, isRequestingSettings, translate, handleToggle, moment, handleSubmitForm, isSavingSettings } = this.props;
+
+		const today = moment(),
+			lastDay = moment( { year: 2017, month: 6, day: 13 } );
+
+		if ( today.isAfter( lastDay, 'day' ) ) {
+			return null;
+		}
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Net Neutrality' ) }>
```

Comment:
That text is good, just changing a little.

> 
The FCC wants to repeal Net Neutrality. Without Net Neutrality, big cable and telecom companies can divide the internet into fast and slow lanes. What would the Internet look like without net neutrality? Find out by enabling this banner on your site: it shows your support for real net neutrality rules by displaying a message on the bottom of your site, and "slowing down" some of your posts.